index.test.js

```diff
@@ -80,19 +91,21 @@ describe('Importer', function() {
 
   test('should support directory imports', function() {
     mockFs.existsSync
-      .mockReturnValueOnce(false) // .scss import
-      .mockReturnValueOnce(true); // directory import
+      .mockReturnValueOnce(true);
 
     expect(importer('~my-module/test', '')).toEqual({
       file: __dirname + '/MOCK_PARENT_DIR/node_modules/my-module/test/index'
     });
   });
 
   test('should support file imports minus extension', function() {
-    mockFs.existsSync.mockReturnValueOnce(true); // .scss import
+    mockFs.existsSync
+      .mockReturnValueOnce(false) // directory import
+      .mockReturnValueOnce(true)  // .scss file import
+    ;
 
     expect(importer('~my-module/test', '')).toEqual({
-      file: __dirname + '/MOCK_PARENT_DIR/node_modules/my-module/test.scss'
+      file: __dirname + '/MOCK_PARENT_DIR/node_modules/my-module/test'
```

Comment:
Will have to check that...

Reply:
What's the difference between "file imports minus extensions" and "file imports"? To me the way the tests are written is a bit confusing, since only the return value of fs.existsSync is changed.
In my opinion it would be easier to provide a mock folder structure and test with that. https://github.com/tschaub/mock-fs provides such functionalities.